readme.txt

```diff
@@ -1,6 +1,6 @@
 === Classic Editor ===
-Contributors: azaozz, melchoyce, chanthaboune, alexislloyd, pento, youknowriad, desrosj, luciano-croce
-Tags: editor, classic editor, block editor, gutenberg
+Contributors: azaozz, luciano-croce, alexislloyd, chanthaboune, desrosj, melchoyce, pento, youknowriad

```

Comment:
The contributor names were added somewhat randomly, think that's good enough? :)

Reply:
OK